TramsDataApi/Controllers/V2/FinancialPlanController.cs

```diff
@@ -80,6 +80,28 @@ public ActionResult<ApiSingleResponseV2<List<FinancialPlanStatus>>> GetAllStatus
 
             return Ok(response);
         }
+        
+        [HttpGet]
+        [Route("closure-statuses")]
```

Comment:
probably follows the current naming convention but should it be 'closure-states' ?